README.md

```diff
@@ -151,6 +152,8 @@ You could use your browser to confirm this. If everything looks OK, you can shar
 
 ### Publish Your Change to Your Github Account
 
+Because our gh-pages branch here is without 'CNAME' file and its configuration has been changed, this gh-pages branch is not suitable for further pull request of merging if you only focus on modifying the content of this website. Instead, we choose to publish all of changes through the 'dev' branch and use the gh-pages branch only for checking those modifications.
```

Comment:
would you mind to add the line like `If you removed CNAME and tweaked _config.yml by following the optional steps` prior to the following session? See this thread for details as well https://github.com/cesehub/cesehub/pull/23#discussion_r70038169